apps/dapp-console-api/src/models/wallets.ts

```diff
@@ -92,6 +94,37 @@ export const getWalletsByEntityId = async (
     .where(and(eq(wallets.entityId, entityId)))
 }
 
+export const getActiveWalletsForEntityByCursor = async (
```

Comment:
not something for this PR! just a though as we start to expand this in the future

I wonder if it will be possible to make this work with generics or just being able to pass in a drizzle table, along with the fields we want to filter on. I think having a single module that we end us using across all of our service and api layers will go along way as our codebases grow

Reply:
good call, I ended up adding a helper to make this work using generics and it should work with any drizzle table. let me know what you think!